src/west/app/project.py

```diff
@@ -274,7 +274,12 @@ def bootstrap(self, args):
         if manifest_project.path:
             manifest_path = manifest_project.path
         else:
-            manifest_path = posixpath.basename(urlparse(manifest_url).path)
+            # We use PurePath() here in case manifest_url is a
+            # windows-style path. That does the right thing in that
+            # case, without affecting POSIX platforms, where PurePath
+            # is PurePosixPath.
+            manifest_path = PurePath(urlparse(manifest_url).path).name
```

Comment:
In case anyone is wondering, it does the right thing with POSIX-style URLs on windows too.